docs/standards/generic-standards/lsp16-universal-factory.md

```diff
@@ -35,11 +35,6 @@ This multi-chain compatibility eliminates the **risk of asset loss** due to mism
 
 ### Specification
 
-:::tip
-
-Check the [**LSP16UniversalFactory contract functions**](../../standards/smart-contracts/lsp16-universal-factory.md) to know how to deploy contracts at the same address across different chains.
```

Comment:
Could you please keep this callout, and redirect to the LSP6 ABI technical docs under the contracts section.

```
:::tip

Check the [**LSP16UniversalFactory contract functions**](../../contracts/contracts/LSP16UniversalFactory/LSP16UniversalFactory.md) to know how to deploy contracts at the same address across different chains.

:::
```

Reply:
fixed